src/RA_AchievementOverlay.h

```diff
@@ -157,43 +144,46 @@ class AchievementOverlay
 public:
     struct NewsItem
     {
-        unsigned int m_nID;
+        unsigned int m_nID{};
         std::string m_sTitle;
         std::string m_sPayload;
-        time_t m_nPostedAt;
+        time_t m_nPostedAt{};
         std::string m_sPostedAt;
         std::string m_sAuthor;
         std::string m_sLink;
         std::string m_sImage;
     };
 
 private:
-    int	m_nAchievementsScrollOffset;
-    int	m_nFriendsScrollOffset;
-    int	m_nMessagesScrollOffset;
-    int	m_nNewsScrollOffset;
-    int	m_nLeaderboardScrollOffset;
-
-    int	m_nAchievementsSelectedItem;
-    int	m_nFriendsSelectedItem;
-    int	m_nMessagesSelectedItem;
-    int	m_nNewsSelectedItem;
-    int	m_nLeaderboardSelectedItem;
-
-    mutable int m_nNumAchievementsBeingRendered;
-    mutable int m_nNumFriendsBeingRendered;
-    mutable int m_nNumLeaderboardsBeingRendered;
-
-    BOOL					m_bInputLock;	//	Waiting for pad release
-    std::vector<NewsItem>	m_LatestNews;
-    TransitionState			m_nTransitionState;
-    float					m_fTransitionTimer;
-
-    OverlayPage				m_Pages[5];
-    unsigned int			m_nPageStackPointer;
-
-    ra::services::ImageReference m_hOverlayBackground;
-    ra::services::ImageReference m_hUserImage;
+    static _CONSTANT_VAR PAGE_TRANSITION_IN{ -0.2F };
+    static _CONSTANT_VAR PAGE_TRANSITION_OUT{ 0.2F };
+
+    int	m_nAchievementsScrollOffset{};
+    int	m_nFriendsScrollOffset{};
+    int	m_nMessagesScrollOffset{};
+    int	m_nNewsScrollOffset{};
+    int	m_nLeaderboardScrollOffset{};
+
+    int	m_nAchievementsSelectedItem{};
+    int	m_nFriendsSelectedItem{};
+    int	m_nMessagesSelectedItem{};
+    int	m_nNewsSelectedItem{};
+    int	m_nLeaderboardSelectedItem{};
+
+    mutable int m_nNumAchievementsBeingRendered{};
+    mutable int m_nNumFriendsBeingRendered{};
+    mutable int m_nNumLeaderboardsBeingRendered{};
+
+    BOOL                  m_bInputLock{}; // Waiting for pad release
+    std::vector<NewsItem> m_LatestNews{};
```

Comment:
In theory, this shouldn't be necessary. Since there was a clearmethod called in the removed Initializemethod, this guarantees it's in a default state.